src/allmydata/scripts/cli.py

```diff
@@ -180,10 +180,21 @@ def parseArgs(self, arg1, arg2=None):
 class PutOptions(FileStoreOptions):
     optFlags = [
         ("mutable", "m", "Create a mutable file instead of an immutable one (like --format=SDMF)"),
-        ]
+    ]
+
     optParameters = [
         ("format", None, None, "Create a file with the given format: SDMF and MDMF for mutable, CHK (default) for immutable. (case-insensitive)"),
-        ]
+
+        ("private-key-path", None, None,
```

Comment:
Maybe also comment that most people can just ignore this option, since they likely don't need it?